ci/requirements-optional-pip.txt

```diff
@@ -14,7 +14,7 @@ lxml
 matplotlib
 nbsphinx
 numexpr
-openpyxl=2.5.5
+openpyxl==2.5.5
```

Comment:
This file is autogenerated. https://github.com/pandas-dev/pandas/issues/22689

Reply:
ok should I roll this edit back?

Reply:
Yes, you should.

Reply:
done